### ISSUE TITLE
[Bug]: Reset zoom. Bug or expected behaviour?

### ISSUE THREAD

**Issue by ux-git** [sircharlo/meeting-media-manager#3139]
### What happened?

The orange Reset button stays even after the image presentation is stopped

### To Reproduce

1. Zoom in to the image
2. Start presenting the media
3. Stop presenting
4. The reset button stays, the zooming level as well.

### What did you expect to happen?

What would be a more expected behaviour? To reset the zooming and hide the orange button, OR keep it as it is now?

### Version

v24.11.0

### What platform are you seeing the problem on?

MacOS

### Screenshots

<img width="137" alt="Screenshot 2024-11-04 at 22 01 05" src="https://github.com/user-attachments/assets/ff2319f4-17cd-4caa-a270-429e81e5cbcc">

**Comment by mtdvlpr** [comment]
I agree. After you've presented the image, the zoom level should be reset.

**Comment by sircharlo** [comment]
Ok! I figured, if you can set the zoom level before presenting, maybe you'd want to keep it after too, but I'll make it reset when stopped if you both think it's more logical.

**Comment by sircharlo** [comment]
a84e789c16b70df91b99be40ae781bb70e0662ab.
